package.json

```diff
@@ -3,19 +3,18 @@
   "version": "0.0.1",
   "description": "OAS (Swagger v3) API Definition for Template API",
   "scripts": {
-    "lint": "node_modules/.bin/speccy lint -s openapi-tags -s operation-tags specification/gp-connect-user-permissions.yaml --skip default-and-example-are-redundant",
-    "publish": "mkdir -p build && node_modules/.bin/speccy resolve specification/gp-connect-user-permissions.yaml -i | poetry run python scripts/yaml2json.py | poetry run python scripts/set_version.py > build/gp-connect-user-permissions.json",
-    "serve": "node_modules/.bin/speccy serve build/gp-connect-user-permissions.json",
+    "lint": "node_modules/.bin/openapi-generator-cli validate -i specification/gp-connect-user-permissions.yaml",
+    "publish": "./node_modules/.bin/openapi-generator-cli generate -i specification/gp-connect-user-permissions.yaml  --generator-key v3.0 -g openapi -o build && rm openapitools.json && mv build/openapi.json build/gp-connect-user-permissions.json",
     "check-licenses": "node_modules/.bin/license-checker --failOn GPL --failOn LGPL"
   },
   "author": "NHS Digital",
   "license": "MIT",
   "homepage": "https://github.com/NHSDigital/gp-connect-user-permissions",
   "dependencies": {
-    "newman-reporter-htmlextra": "^1.20.1",
-    "speccy": "^0.11.0"
+    "newman-reporter-htmlextra": "^1.20.1"
```

Comment:
I don't think we need this dependency. It's for postman tests which we don't use. Please remove it otherwise it'll generate annoying dependabot warnings.

Reply:
This dependency was part of repo before, so i didn't changed it. Let me now remove it